.github/workflows/node-versions.yml

```diff
@@ -0,0 +1,39 @@
+name: Test build on different node versions on all platforms
+
+on:
+  schedule:
+  - cron: '0 12 * * 0'
+
+  workflow_dispatch:
+
+jobs:
+
+  test-node-10:
+    uses: ./.github/workflows/node-test.yml
+    secrets: inherit
+    with:
+      gh-node-version: '10.x'
+
+  test-node-12:
+    uses: ./.github/workflows/node-test.yml
+    secrets: inherit
+    with:
+      gh-node-version: '12.x'
+
+  test-node-14:
+    uses: ./.github/workflows/node-test.yml
+    secrets: inherit
+    with:
+      gh-node-version: '14.x'
+
+  test-node-16:
+    uses: ./.github/workflows/node-test.yml
+    secrets: inherit
+    with:
+      gh-node-version: '16.x'
+
+  test-node-18:
+    uses: ./.github/workflows/node-test.yml
+    secrets: inherit
+    with:
+      gh-node-version: '18.x'
```

Comment:
A few questions here:

1. Do we want to test node 10.x? it's pretty old...
2. Do we know of a method to test `17.x`, or other common, but non-LTS, versions? I did a (very) quick search and didn't find anything.
3. Do we want to show node versions badge successf/fail in an aggregated manner?
   * i.e. if nodev14 only fails on windows, should the badge be red? If not, the work involved in that is significant, so maybe a separate issue could track that decoupling.

Reply:
1.  let's remove `10.x` 
   - when we make things build on  current LTS (`16.x`) we should also remove `12.x` and `14.x`
2. I think you can use  `lts/*` and `current`  to always run against latest versions from  https://nodejs.org
3.  No need to be that verbose, if job fails then one can go to action and see what failed.